tests/test_project/apps.py

```diff
@@ -1,18 +1,33 @@
 from django.apps import AppConfig
 from django.conf import settings
 
+from openwisp_utils import settings as app_settings
+
 
 class TestAppConfig(AppConfig):
     name = 'test_project'
     label = 'test_project'
 
+    _default_drf = {
+        'DEFAULT_THROTTLE_CLASSES': ['rest_framework.throttling.ScopedRateThrottle'],
+        'DEFAULT_THROTTLE_RATES': {
+            'anon': app_settings.API_RATE_ANON,
+        },
+    }
+
     def ready(self, *args, **kwargs):
         super(TestAppConfig, self).ready(*args, **kwargs)
         self.add_default_menu_items()
+        self.configure_drf_defaults()
 
     def add_default_menu_items(self):
         menu_setting = 'OPENWISP_DEFAULT_ADMIN_MENU_ITEMS'
         items = [
             {'model': 'test_project.Shelf'},
         ]
         setattr(settings, menu_setting, items)
+
+    def configure_drf_defaults(self):
```

Comment:
I'm also thinking that this method could be part of a mixin which we could import in the other modules so we don't need to redefine it each time, we will just add the mixin to the AppConfig class, or alternatively we can define a function and call that function in the modules (instead of using the mixin, which some people don't like, I'm fine with both approaches).